packages/test/.expo-shared/assets.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
whats this?

Reply:
This is a generated file. I did not touch it.